src/Microsoft.Data.SqlClient/netfx/src/Resources/Strings.Designer.cs

```diff
@@ -9224,12 +9224,11 @@ internal class Strings {
                 return ResourceManager.GetString("SQL_IntegratedWithUserIDAndPassword", resourceCulture);
             }
         }
-
+        
```

Comment:
Since this is an autogenerated file, editing manually must be not introduce changes that the generator would not introduce or else the next unrelated update to the resx will undo these changes.